controlplane/test/utils.test.ts

```diff
@@ -18,4 +18,15 @@ describe('Utils', () => {
     expect(normalizeLabelMatchers(['A=value2,B=value', 'B=value'])).toEqual(['A=value2,B=value', 'B=value']);
     expect(normalizeLabelMatchers(['A=value,B=value', 'A=value,B=value'])).toEqual(['A=value,B=value']);
   });
+
+  test('mergeURLS', () => {
+    expect(mergeUrls('http://example.com', 'path')).toBe('http://example.com/path');
+    expect(mergeUrls('http://example.com', '/path')).toBe('http://example.com/path');
+    expect(mergeUrls('http://example.com/', 'path')).toBe('http://example.com/path');
+    expect(mergeUrls('http://example.com/', '/path')).toBe('http://example.com/path');
```

Comment:
This seems like a repeat of the previous lines

Reply:
No, the second one has slash in the start of the relative path